.ci/common.sh

```diff
@@ -31,6 +22,11 @@ REPOSITORY=${VCS}/${ORGANIZATION}/${PROJECT}
 # The `go <cmd>` commands requires to see the target repository to be part of a
 # Go workspace. Thus, if we are not yet in a Go workspace, let's create one
 # temporarily by using symbolic links.
+if [[ -z "$SOURCE_PATH" ]]; then
```

Comment:
is this check required, considering that you source this script from `.ci/build` where you export SOURCE_PATH. Also you have set `set -eo pipefail` which will already exit if there was an issue setting this .

Reply:
`common.sh` is a bash script that can still be run independent of the other `.ci` scripts, so keeping this check made sense. Also, if an importer of common.sh does not set `SOURCE_PATH`, then this check allows us to fail early and with a meaningful error too.